src/helpers/helpers.ts

```diff
@@ -0,0 +1,103 @@
+import { SortType } from '../enums/SortType';
+import { Person } from '../types';
+
+export const findParent = (
+  people: Person[],
+  parentName: string | null,
+): Person | null => {
+  return people.find(({ name }) => name === parentName) || null;
+};
+
+export const getTypeOfSort = (sortType: string | null): SortType => {
+  switch (sortType) {
+    case SortType.NAME:
+      return SortType.NAME;
+
+    case SortType.SEX:
+      return SortType.SEX;
+
+    case SortType.BORN:
+      return SortType.BORN;
+
+    case SortType.DIED:
+      return SortType.DIED;
+
+    case SortType.NONE:
+    default:
+      return SortType.NONE;
+  }
+};
+
+export const filterPeople = (
+  people: Person[],
+  sex: string | null,
+  query: string | null,
+  centuries: string[],
+): Person[] => {
+  let peopleFiltered = [...people];
```

Comment:
Method `filter` returns a new array every time, so you don’t need to copy it with spread operator.